lib/views.js

```diff
@@ -17,21 +17,48 @@
  \*───────────────────────────────────────────────────────────────────────────*/
 'use strict';
 
+var path = require('path');
+var assert = require('assert');
 var thing = require('core-util-is');
 var debug = require('debuglog')('kraken/views');
+function tryResolverHelper(paths=module.paths) {
+    // If this module is deployed outside the app's node_modules, it wouldn't be
+    // able to resolve plugin modules deployed under app. Adding app's node_modules 
+    // folder to the paths will help handle this case.
+    const appDependencyPath = path.resolve(process.cwd(), 'node_modules');
+    if (!paths.includes(appDependencyPath)) {
+        paths = [ ...paths ];
+        paths.push(appDependencyPath);
+    }
+    return function tryResolve(modulePath) {
+        assert.strictEqual(typeof modulePath, 'string', 'modulePath should be a string');
+        try {
+            return require.resolve(modulePath, { paths });
+        } catch (e) {
+            if (e.code === 'MODULE_NOT_FOUND')
+                return;
```

Comment:
when will this happen? and why are we silencing this error?

Reply:
never mind. I see you have a check in Line 58

Reply:
It will throw later `throw new TypeError(`Module ${spec.module} not found`);` LINE#59